build.gradle

```
@@ -92,3 +95,18 @@ task printCurrentVersion {
     println version
   }
 }
+
+jacocoTestReport {
```

デフォルトだとjacocoTestReport taskはtest taskへの依存がなく、最後に実行されたテスト結果をもとにカバレッジを計算するようです
なんだか気持ち悪いので、test taskへの依存を足しませんか
gradleがよしなにしてくれるのでCIでtestが2回実行されるようなことはないはずです

```suggestion
jacocoTestReport {
  dependsOn test
```

---

> CIでtestが2回実行される

まさにこれが嫌で依存関係は定義していなかったんだけど

> gradleがよしなにしてくれる

これは知りませんでした．手元でやってみると確かに 2 回実行されることはありませんでした．